src/PaymentGateways/PayPalStandard/Views/PayPalStandardBillingFields.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Give\PaymentGateways\PayPalStandard\Views;
+
+use Give\Helpers\Form\Utils;
+
+/**
+ * This class handle donation form billing field markup for PayPal Standard.
+ *
+ * @unlreased
+ */
+class PayPalStandardBillingFields
+{
+    /**
+     * @unlreased
+     *
+     * @param int $formId
+     *
+     * @return bool|void
+     */
+    public function __invoke($formId)
+    {
+        if (give_is_setting_enabled(give_get_option('paypal_standard_billing_details'))) {
+            give_default_cc_address_fields($formId);
+
+            return true;
+        }
+
+        if (Utils::isLegacyForm($formId)) {
+            return false;
+        }
+
+        printf(
+            '
+            <fieldset class="no-fields">
+                <div style="display: flex; justify-content: center; margin-top: 20px;">
+                    <svg width="250" height="66" viewBox="0 0 250 66" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I would suggest moving the SVG markup to a `resources/templates` directory or something, since the markup is so large (and the specific markup isn't important here in regards to abstractions).

Reply:
fixed: https://github.com/impress-org/givewp/compare/eeba557da48c...55949ae3613b